currencies.yml

```diff
@@ -0,0 +1,1544 @@
+---
+NIO:
+  minor_unit: 2
+  currency: Cordoba Oro
+  symbol: C$
+  tags:
+  - :iso4217
+  - :oxr
```

Comment:
Rather than OXR being a tag, I feel we should have a `supported_providers` array with `:oxr` and `:ecb` as possibilities.